autoprof/fit/base.py

```diff
@@ -124,4 +125,4 @@ def _f(x, nu):
 
             if res.success:
                 return res.x[0]
-        raise RuntimeError(f"Unable to compute Chi^2 contour for ndf: {ndf}")
+        raise RuntimeError(f"Unable to compute Chi^2 contour for n_params: {n_params}")
```

Comment:
`ndf` is standard terminology for "number of degrees of freedom", while `n_params` is also correct, `ndf` is the standard and I think we should keep it.

Reply:
Okay...